mlos_bench/mlos_bench/tests/config/schemas/services/test-cases/good/partial/azure_auth_service-partial.jsonc

```diff
@@ -0,0 +1,4 @@
+{
+    "class": "mlos_bench.services.remote.azure.AzureAuthService",
+    "config": {}
```

Comment:
Would be cool to omit the "config" attr in that case too.

Reply:
```suggestion
```
let's do it :)

Reply:
Do you want me to leave a PR against yours on that?

Reply:
actually, removing that line would require changing lots of schema files.. let's do it against master in the next PR